Makefile

```diff
@@ -57,15 +57,17 @@ src/curr.rs: $(XDR_FILES_LOCAL_CURR)
 		gem specific_install https://github.com/stellar/xdrgen.git -b master && \
 		xdrgen --language rust --namespace curr --output src/ $^ \
 		'
-	rustfmt src/curr.rs
+	sudo chown $$USER $@
```

Comment:
I want to fix this issue by changing the docker command, so that we aren't running sudo on the host. Let me circle back on that.

Reply:
It appears to be phenomenally complex to make this work. https://jtreminio.com/blog/running-docker-containers-as-current-host-user/

I'm a bit surprised, but not really surprised.

Reply:
This should do it. I tested it on Ubuntu. Let me know if it doesn't fix it for you. https://github.com/stellar/rs-stellar-xdr/pull/71